utils/tag_and_transmit.py

```diff
@@ -72,15 +76,19 @@ def run(
 
             if isinstance(SOPClassUID, type("pydicom.uid.UID")):
                 # Transmit DICOM file to server specified
-                dicom_transmitted = transmit_dicom_file(
-                    path, destination, called_ae, port, calling_ae
-                )
+                try:
+                    dicom_transmitted = transmit_dicom_file(
+                        path, destination, called_ae, port, calling_ae
+                    )
+                except TemporaryFailure:
+                    log.error('Could not export '+path.name)
```

Comment:
what about logging that this is because of congestion failure ?

Reply:
There is already a line that does that: https://github.com/flywheel-apps/dicom-send/pull/19/files#diff-e3448bdb7993be9eb2997ca7024216fba05c1d4399e7b1a39a37a4eab77a2afcR138